webview/src/plots/components/comparisonTable/ComparisonTableRow.tsx

```diff
@@ -128,21 +134,47 @@ export const ComparisonTableRow: React.FC<ComparisonTableRowProps> = ({
             className={cx({ [styles.pinnedColumnCell]: pinnedColumn })}
             colSpan={pinnedColumn ? 1 : nbColumns}
           >
-            <div className={styles.rowPath}>
-              <button
-                className={styles.rowToggler}
-                onClick={toggleIsShownState}
-              >
-                <Icon icon={isShown ? ChevronDown : ChevronRight} />
-                <Tooltip
-                  content={path}
-                  placement="bottom-start"
-                  delay={NORMAL_TOOLTIP_DELAY}
+            <div className={styles.rowHeader} data-testid="row-header">
+              <div className={styles.rowHeaderPath}>
+                <button
+                  className={styles.rowToggler}
+                  onClick={toggleIsShownState}
                 >
-                  <span className={styles.pathText}>{path}</span>
-                </Tooltip>
-              </button>
-              <CopyButton value={path} className={styles.copyButton} />
+                  <Icon icon={isShown ? ChevronDown : ChevronRight} />
+                  <Tooltip
+                    content={path}
+                    placement="bottom-start"
+                    delay={NORMAL_TOOLTIP_DELAY}
+                  >
+                    <span className={styles.pathText}>{path}</span>
+                  </Tooltip>
+                </button>
+                <CopyButton value={path} className={styles.copyButton} />
+              </div>
+              {boundingBoxClassesArr.length > 0 && (
+                <div className={styles.boundingBoxClasses}>
+                  <p className={styles.boundingBoxClassesTitle}>Classes</p>
+                  {boundingBoxClassesArr.map(([label, { color, selected }]) => (
+                    <React.Fragment key={label}>
+                      <input
+                        type="checkbox"
+                        id={color.slice(1)}
+                        name="labels"
+                        value={label}
+                        defaultChecked={selected}
+                        className={styles.hiddenInput}
+                      />
+                      <label
+                        className={styles.boundingBoxClassesButton}
+                        style={{ background: color }}
```

Comment:
Can we ever get a background color on which to foreground color makes the text not accessible? If so, we can maybe calculate whether it should use a dark or light foreground color.

Reply:
If I understood the libraries that I got the colors from, they always use white as the text color. While testing, while I think a couple colors could benefit from a darker color, white 
does seem to work well enough:

![Screenshot 2024-01-25 at 8 29 08 AM](https://github.com/iterative/vscode-dvc/assets/43496356/06560d95-c6f9-49e7-830e-12aeaf31844a)

Reply:
Yes, apart from the neon green they all look fine. And I don't think there is anything we can do about that neon green anyway.